Documentation/Books/Manual/Programs/Arangoinspect/Examples.md

```diff
@@ -0,0 +1,116 @@
+Arangoinspect Examples
+======================
+
+If you are asked by ArangoDB support to provide an `arangodb-inspector.json`
+file, run the _arangoinspect_ binary to generate in the current working folder.
+
+The resulting JSON file contains the names of existing databases, their
```

Comment:
The resulting JSON file is a collection of meta data acquired from all involved instances. The data includes relevant operating system parameters, arangodb process parameters, local database information etc.